diesel_derives2/src/insertable.rs

```diff
@@ -0,0 +1,82 @@
+use syn;
+use quote;
+use proc_macro2::Span;
+
+use field::*;
+use model::*;
+use util::*;
+
+pub fn derive(item: syn::DeriveInput) -> Result<quote::Tokens, Diagnostic> {
+    let model = Model::from_item(&item)?;
+
+    if model.fields().is_empty() {
+        return Err(Span::call_site()
+            .error("Cannot derive Insertable for unit structs")
+            .help(format!(
+                "Use `insert_into({}::table).default_values()` if you want `DEFAULT VALUES`",
```

Comment:
How does this help me when my structure fields are missing? What is the use case?

Reply:
I'm assuming that if you derive this on a struct with no fields, you're likely trying to do `INSERT DEFAULT VALUES`. Otherwise I'm not sure what they're trying to accomplish.